presets/massa-station-preset.js

```diff
@@ -83,19 +83,19 @@ module.exports = {
           fontSize: "16px",
           fontWeight: "600",
           fontFamily: theme("fontFamily.Urbane"),
-          lineHeight: "19px",
+          lineHeight: "20px",
```

Comment:
Hey Lucas, these colors changes should be in a specific MR for that.

We are dealing here with the selector and we are changing colors in the same MR.
So, IMHO we should have one MR to change the colors. Then, you can go further with your component.

Reply:
changes have been manually reverted so the new line height can be addressed in specific MR !

Reply:
Great! Tks!